packages/app/cypress/support/index.js

```diff
@@ -13,5 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import '@bigtest/cypress';
```

Comment:
If I remember correctly, importing `@bigtest/cypress` in here shouldn't be necessary. The `cy.expect()` and `cy.do()` commands are automatically "registered" when you import a bigtest module in your teset.

Reply:
I use interactors, and they don't depend on bigtest/cypress

Reply:
ohh i see. right that makes sense. 👌